### PR TITLE
feat: Admin app category mgmt

### PR DESCRIPTION
**Pull request by iwisunny**

### CODE REVIEW

File: src/components/Layout/SideNav/index.scss [hunk 1]
```diff
@@ -1,5 +1,11 @@
 @import '~scss/vars';
 
+%disabledLink {
+  color: $N75;
+  opacity: 0.5;
+  cursor: not-allowed;
+}
+
 .nav {
```
Comment (whDongRui) [inline src/components/Layout/SideNav/index.scss]: Suggest this style can remove _mixins.scss file, if do this other file can reference.

File: src/stores/app/index.js [hunk 2]
```diff
@@ -165,19 +162,24 @@ export default class AppStore extends Store {
       delete params.noLoading;
     }
 
-    const result = await this.request.get(
-      'apps',
-      assign(defaultParams, params)
-    );
-
+    const result = await this.request.get('apps', params);
     const apps = get(result, 'app_set', []);
```
Comment (whDongRui) [inline src/stores/app/index.js]: If this method tranfer params is same the store parameter like search_word, owner, the store define variable will cover the transfer parameter。

File: src/components/AppsTable/index.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import "~scss/vars";
```
[the remark this answers is not in the document]
Comment (iwisunny) [reply src/components/AppsTable/index.scss]: ok

File: src/components/Layout/SideNav/index.jsx [hunk 4]
```diff
@@ -336,15 +330,16 @@ class SideNav extends React.Component {
...
-                  name={nav.iconName}
-                  type={this.isLinkActive(nav.active) ? 'light' : 'dark'}
-                />
-              </Link>
+              <Icon
+                className={styles.icon}
+                size={20}
+                name={nav.iconName}
+                type={this.isLinkActive(nav.active) ? 'light' : 'dark'}
+              />
```
[the remark this answers is not in the document]
Comment (iwisunny) [reply src/components/Layout/SideNav/index.jsx]: I think when click icon, no need to change route

File: src/components/Layout/SideNav/index.scss (hunk 1 above)
Comment (iwisunny) [reply src/components/Layout/SideNav/index.scss]: ok

File: src/stores/app/index.js (hunk 2 above)
Comment (iwisunny) [reply src/stores/app/index.js]: user defined params have higher priority

**Pull request merged**